MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -88,4 +88,31 @@ protected virtual void NavigateToFirstViewModel(object hint)
             }
         }
     }
+
+    public class MvxAppStart<TViewModel, TParameter> : MvxAppStart<TViewModel> where TViewModel : IMvxViewModel<TParameter>
+    {
+        public MvxAppStart(IMvxApplication application, IMvxNavigationService navigationService) : base(application, navigationService)
+        {
+        }
+
+        protected override void NavigateToFirstViewModel(object hint)
+        {
+            TParameter navParam = default;
+
+            hint = Application.GetAppStartHint(hint);
```

Comment:
This line should be something like `hint = hint ?? Application.GetAppStartHint(hint);`. Otherwise the parameter is always replaced and there's no way to pass anything from the native platform

Reply:
I'm not sure I agree with this as the app start hint can be additive, especially when using a MvxBundle.

Reply:
Also, notice that the default implementation is to return the provided hint, so nothing is lost.

Reply:
You're right! Sorry